torch_glow/tests/utils.py

```diff
@@ -92,15 +92,23 @@ def assert_equivalent(result, other_result, atol=5e-4, rtol=1e-3):
             assert_equivalent(a, b, atol=atol, rtol=rtol)
             for a, b in zip(result, other_result)
         )
-    elif torch.allclose(result, other_result, atol, rtol):
-        return True
+    elif other_result.dtype == torch.bool:
+        if torch.all(torch.eq(result, other_result)):
+            return True
+        else:
+            diff = ~(other_result < result)
```

Comment:
In this way the diff would be either -1 or -2 (please correct me if I'm wrong). Could we use the result of torch.eq() as the diff and print it.

Reply:
Good point, I made the change you suggested.